Pipfile.lock

```diff
@@ -1070,50 +1080,50 @@
         },
         "invenio-oaiserver": {
             "hashes": [
-                "sha256:70f16e872da33251a50b5e3b83eaa2e4ffd2b695d36c4fb3a8ddb1e70d825d0e",
-                "sha256:73cb845e50a2b709cebf5beceb751c7e0dd21748f7f0a51e20c7b4695ff8a1a7"
+                "sha256:31648f2c62de0e3819da5dc8e7c8987a43000deb5c538c3be37f227da9b0d476",
+                "sha256:8342979ccb70d64ab1160d817e750f031a437dcbcefbba4857c7d611a396d087"
             ],
             "markers": "python_version >= '3.7'",
-            "version": "==2.1.1"
+            "version": "==2.2.0"
         },
         "invenio-oauth2server": {
             "hashes": [
-                "sha256:2e86c0a98843218aa5a3924dcbd46f0b25baa735d3410aaee73ad8bcb3286d72",
-                "sha256:c35c46d0c2bdf644e5730438d763fc3e7695a66d3a4a4ffe1bb215e196ff7339"
+                "sha256:40466fadb70ec02dd4e86c650e88799fe639cb245f99cbca3f4de0fdf2836505",
+                "sha256:95898ddd321b8e6ad58fd7cb6cd11c116bdbd5d141bb2d97fc8422072310862e"
             ],
             "markers": "python_version >= '3.7'",
-            "version": "==1.3.8"
+            "version": "==2.0.0"
         },
         "invenio-oauthclient": {
             "hashes": [
-                "sha256:1b79eecc4c762decd63ee08928a433d177c01ecba1ede3f709a1468076e79b83",
-                "sha256:d4db0ddecf158702d894fc6089a6f85171e27243aacb9081fe12295b8f56fdbe"
+                "sha256:736a1703b334f71da43813d3c0701a3ccf62023eaa20222e71e07ef83602b3b1",
+                "sha256:85fffbda045f0459558866d7f6b84a54e8fcf92e63d6a5b92c95946ac0cb6a06"
             ],
             "markers": "python_version >= '3.7'",
-            "version": "==2.1.1"
+            "version": "==2.3.0"
```

Comment:
With this change, you will need to change the signature of the info serializer func. [This](https://github.com/CERNDocumentServer/cds-rdm/blob/master/site/cds_rdm/oidc.py#L19) is old, it should match [this](https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/contrib/keycloak/handlers.py#L48) when upgrading `invenio-oauthclient`.
I can test the login locally if you prefer.

Reply:
Fixed and tested locally, works fine.